cookbook/examples/apps/whatsapp_chat_agent/requirements.txt

```diff
@@ -0,0 +1,10 @@
+fastapi>=0.68.0
+uvicorn>=0.15.0
+python-dotenv>=0.19.0
+requests>=2.26.0
+yfinance>=0.1.63
+openai>=1.0.0
+agno>=0.1.0
```

Comment:
```suggestion
agno>=1.1.9
```

Reply:
Would actually have to be 1.1.10 after we release this, so the agent would have the right tools. Maybe don't specify a version, then it would be latest.